### PR TITLE
feat: added thumbnail preview in desktop

### PR DESCRIPTION
**Pull request by Sardar1208**
**- What I did - added thumbnail for videos in desktop**
ticket link - https://github.com/atsign-foundation/apps/issues/787

### CODE REVIEW

**Comment by sachins-geekyants** [comment]
Are we using `thumblr ` to generate thumbnails for all platforms ?

File: lib/desktop_screens_new/my_files_screen/widgets/category_screen.dart [hunk 1]
```diff
@@ -115,6 +115,15 @@ class _CategoryScreenState extends State<CategoryScreen> {
...
           children: [
+            InkWell(
+              onTap: () async {
+                await generateVideoThumbnail(
+                    "C:\\Users\\Dell\\Documents\\@mosphere-pro\\@snowboard89\\Screen_Recording_2023-08-22_at_2.28.28_PM.mov");
+              },
+              child: Container(
+                child: Text("generate"),
+              ),
+            ),
```
Comment (sachins-geekyants) [inline lib/desktop_screens_new/my_files_screen/widgets/category_screen.dart]: Do we need this change ?

File: lib/screens/my_files/widgets/recents.dart [hunk 2]
```diff
@@ -222,3 +249,31 @@ Future videoThumbnailBuilder(String path) async {
...
+
+Future<dynamic> generateVideoThumbnail(String path) async {
+  final plugin = FcNativeVideoThumbnail();
+
+  String thumbnailPath = path;
+  var temp = thumbnailPath.split("\\");
+  var fileNamewithExt = temp.removeLast();
+  var parentPath = temp.join("\\");
+
+  var file = File("${parentPath}\\${fileNamewithExt}_thumbnail.jpeg");
```
Comment (sachins-geekyants) [inline lib/screens/my_files/widgets/recents.dart]: Should there have been `Platform.separator` instead of `\\` ?

Comment (sachins-geekyants) [reply lib/screens/my_files/widgets/recents.dart]: Windows and macos uses different separator for file location.

File: lib/screens/my_files/widgets/recents.dart [hunk 3]
```diff
@@ -222,3 +249,31 @@ Future videoThumbnailBuilder(String path) async {
...
+    srcFile: path,
+    destFile: "${parentPath}\\${fileNamewithExt}_thumbnail.jpeg",
+    width: 300,
+    height: 300,
+    keepAspectRatio: true,
+    format: 'jpeg',
+    quality: 90,
+  );
+
+  print(thumbnailGenerated);
```
Comment (sachins-geekyants) [inline lib/screens/my_files/widgets/recents.dart:274]: print can be removed.

**Review by sonle-geekyants — APPROVED** [review]

**Pull request merged**